packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx

```diff
@@ -15,12 +15,82 @@ import {
 } from '@lightdash/common';
 import React, { FC, useCallback, useMemo, useState } from 'react';
 import { useOrganisation } from '../../hooks/organisation/useOrganisation';
+import { useTracking } from '../../providers/TrackingProvider';
+import { EventName } from '../../types/Events';
 import { useVisualizationContext } from '../LightdashVisualization/VisualizationProvider';
-import { InputWrapper, Wrapper } from './ChartConfigPanel.styles';
+import {
+    InputWrapper,
+    MinMaxContainer,
+    MinMaxInput,
+    MinMaxWrapper,
+    Wrapper,
+} from './ChartConfigPanel.styles';
 import FieldLayoutOptions from './FieldLayoutOptions';
 import BasicSeriesConfiguration from './Series/BasicSeriesConfiguration';
 import GroupedSeriesConfiguration from './Series/GroupedSeriesConfiguration';
 
+interface MinMaxProps {
+    minVal: (index: number, value: string | number | undefined) => void;
+    maxVal: (index: number, value: string | number | undefined) => void;
+    index: number;
+    axis: string;
+}
+
+const AxisMinMax: FC<MinMaxProps> = ({ minVal, maxVal, index, axis }) => {
```

Comment:
What you think if this component was "dumber" ? it wouldn't depend on `useVisualizationContext` or have axis logic: 
```
interface MinMaxProps {
    label: string;
    min:  string | number | undefined;
    max:  string | number | undefined;
    setMin: (index: number, value: string | number | undefined) => void;
    setMax: (index: number, value: string | number | undefined) => void;
}
```
Use:

```
<AxisMinMax
         label={`Auto ${dirtyLayout?.flipAxes ? 'X' : 'Y'}-axis range (${dirtyLayout?.flipAxes ? 'bottom' : 'left'})`}
         min={dirtyEchartsConfig?.yAxis?.[0]?.min}
         max={dirtyEchartsConfig?.yAxis?.[0]?.max}
         setMin={setYMinValue}
         setMax={setYMaxValue}
/>
```

Reply:
The AxisMinMax component shouldn't care about the index if is a ""dumb" component, that should be applied outside 

```
<AxisMinMax
         label={`Auto ${dirtyLayout?.flipAxes ? 'X' : 'Y'}-axis range (${dirtyLayout?.flipAxes ? 'bottom' : 'left'})`}
         min={dirtyEchartsConfig?.yAxis?.[0]?.min}
         max={dirtyEchartsConfig?.yAxis?.[0]?.max}
         setMin={(newValue: string | undefined ) => setYMinValue(0, newValue)}
         setMax={(newValue: string | undefined ) => setYMaxValue(0, newValue)}
/>
```